README.md

```diff
@@ -231,13 +233,11 @@ console.log(result);
 </pre>
 ```
 
-That is all for the live example piece. All that remains is to tell the page generator about our new example. To do this, open up `site.json` at the root of the project folder. Under `pages`, find an exising entry with the `type` of `js`.
-
-Copy and paste an example.
+That is all for the live example piece. All that remains is to tell the page generator about our new example. To do this, open up `site.json` at the root of the project folder. Under `pages`, find an existing entry with a `type` of `js`.
 
-> The file is ordered by `type`, and then alphabetically for each `type`.
+Copy and paste the example then update it to apply to your new example, noting that pages are grouped by `type`, and then alphabetically for each `type`.
```

Comment:
Should this "Copy and paste the example then update i" be "Copy and paste the example, then update i"